ProcessLib/SurfaceFlux/SurfaceFluxLocalAssembler.h

```diff
@@ -120,13 +120,33 @@ class SurfaceFluxLocalAssembler final
                        std::vector<GlobalVector*> const&)>
                        getFlux) override
     {
-        auto surface_element_normal =
-            MeshLib::FaceRule::getSurfaceNormal(&_surface_element);
-        surface_element_normal.normalize();
-        // At the moment (2016-09-28) the surface normal is not oriented
-        // according to the right hand rule
-        // for correct results it is necessary to multiply the normal with -1
-        surface_element_normal *= -1;
+        auto getSurfaceNormal =
```

Comment:
This is a variable, should be `get_surface_normal`.
```suggestion
        auto get_surface_normal =
```

Reply:
```c++
struct { void operator() { } } get_surface_normal;
```